nginx.master.conf

```diff
@@ -4,26 +4,18 @@ include common/main.conf;
 http {
     # Without this, cosocket-based code in worker
     # initialization cannot resolve leader.mesos.
-    resolver 127.0.0.1:61053;
+    resolver 127.0.0.1:61053 ipv6=off;
```

Comment:
We're not doing IPv6 so this will save us some queries to DNS backend that would get empty replies anyways.